packages/terra-action-header/src/ActionHeader.module.scss

```diff
@@ -9,6 +9,11 @@
     margin-left: var(--terra-action-header-margin-left, 1.429rem);
   }
 
+  .left-buttons,
+  .right-buttons {
+    padding-top: var(--terra-action-header-button-padding-top);
```

Comment:
If feels a little strange to use padding to calculate an offset to center content. If this component is using flexbox then using something like align-items center may have a similar centering effect without needing to calculate these offsets. There's a few ways to center content without padding, flexbox is just an example.

Reply:
I agree using align-items is more appropriate solution to align buttons to center. since action-header had already align-items set to flex-start changing this to center will create minor discrepancy in existing action-header. So I went with using padding value to align action buttons to center like [Title](https://github.com/cerner/terra-core/blob/1061a1a9d6e60ab314a451a4465d30fdfa57020f/packages/terra-action-header/src/ActionHeaderContainer.module.scss#L52).

Reply:
Reviewing UX needs for theme and best approach.